engines/python/setup/djl_python/rolling_batch/scheduler_rolling_batch.py

```diff
@@ -98,11 +98,18 @@ def _init_model_and_tokenizer(self,
                 "ForConditionalGeneration"):
             raise ValueError('Seq2Seq model is not supported by scheduler')
         else:
+            device_map = "cpu"
+            if device:
+                if isinstance(device, str):
+                    device_map = device
```

Comment:
can you pass device_map as 0 or 1 when we consider different GPU?

Reply:
no, there could be a scenario that the input device is the specific GPU ID, did we offer anything could work on this value?

Reply:
device=0

Reply:
anyway, we can leave this later since DJLServing serving layer could cover it. Could you add a TODO in here to support specific GPU IDs?